docs/core/porting/project-structure.md

```diff
@@ -63,7 +63,7 @@ Changes to note are:
 
 ## See also
 
-* Please see the [.NET Core porting documentation][porting-doc] for more guidance on migrating to .NET Core.
+Please see the [.NET Core porting documentation][porting-doc] for more guidance on migrating to .NET Core.
 
 [porting-doc]: index.md
```

Comment:
These links don't work at all.

Reply:
Why not merge the links from line 68 to line 80 into the article?

Reply:
Why not inline the links from line 68 to line 80?

Reply:
Yeah, you can do that. I actually prefer inline. I think the PM who wrote this liked this style of link.